astro/src/pages/docs/quickstarts/quickstart-sections.ts

```diff
@@ -42,13 +42,6 @@ const qsSections: QuickStartSection[] = [
         faIcon: 'fa-droplet',
         navColor: 'blue',
       },
-      {
-        href: '/docs/v1/tech/tutorials/integrate-expressjs',
```

Comment:
My understanding is that when the new quickstart drops in that it will replace the existing and get picked up by

```
quickstarts.filter(quickstart => quickstart.id.indexOf("index.md") === -1)
  .map(qs => (
    {
      href: `/docs/quickstarts/${qs.slug}/`,
      title: qs.data.title,
      icon: qs.data.icon,
      faIcon: qs.data.faIcon,
      navColor: qs.data.color,
      ...qs
    }))
  .forEach(qs => qsSections.find(section => section.key === qs.data.section).articles.unshift(qs));
  ```